benchmarks/trace/grpc/service.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package benchmarks;
+
+service Pinger {
+  rpc PingUniTraced(Ping) returns (Pong) {}
```

Comment:
What is Uni?

PingTraced?

Reply:
Uni -- one direction, so not streaming.

Reply:
Yeah am going to remove this, and make it your suggestion.